src/AWSSQS.jl

```diff
@@ -20,11 +20,11 @@ export sqs_list_queues, sqs_get_queue, sqs_create_queue, sqs_delete_queue,
 
 
 using AWSCore
-using SymDict
+using AWSSDK.SQS
```

Comment:
AWSSQS.jl does not need AWSSDK.jl
It calls `AWSCore.Services.sqs`, defined here: https://github.com/JuliaCloud/AWSCore.jl/blob/master/src/Services.jl#L1572